src/hooks/useGetArticle.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticle } from "../api/fetch";
+
+export default function useGetArticle({ match }) {
```

Comment:
This hook has a dependency to ReactRouters `match`  and a form now. 
Better isolate it and add the `articleId` and the updated `article` (data) as arguments. 
With this change, you can use this hook in more scenarios like "I like to load an old state of an article"

Reply:
Thanks for the feedback, i will try to do it. :)

Reply:
Don't know if i did it right now, but i tried to split the code a bit more. I have in mind that we're already talking about loading an old state, but do you have an example for this? Cause now i didn't send `article` (data) as argument to the `useGetArticle` hook.